<launch>
    <!-- <param name="use_sim_time" value="true"/>; -->

    <arg name="path" default="/home/dearmoon/datasets/NWU/"/>
    <arg name="scene_0" default="日晴不颠簸低速3"/>
    <arg name="scene_1" default="日雪不颠簸高速"/>
    <arg name="scene_2" default="夜雪不颠簸高速"/>

    <arg name="scene" default="$(arg scene_2)"/>

    <node name="pc2ToCustomMsg" type="pc2ToCustomMsg" pkg="rosbag_tools" output="screen">
        <param name="outputFile" value="$(arg path)$(arg scene)/enhancing_v2/1/CustomMsg.bag"/>
    </node>

    <node name="player" pkg="rosbag" type="play" 
        args="--clock
        $(arg path)$(arg scene)/enhancing_v2/1/regionGrowingResult.bag"
        >
    </node>

</launch>